Add support for armv6 and armv7

I am building an app for multiple platforms. One of that platform is a Raspberry Pi.
These use armv8. I am trying to run libsql on armv7 in qemu to test the package and i got this error
```
node:internal/modules/cjs/loader:1244
  const err = new Error(message);
              ^

Error: Cannot find module '@libsql/linux-arm-gnueabihf'
Require stack:
- /app/server/node_modules/libsql/index.js
    at Function._resolveFilename (node:internal/modules/cjs/loader:1244:15)
    at Function._load (node:internal/modules/cjs/loader:1070:27)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
    at Module.require (node:internal/modules/cjs/loader:1335:12)
    at require (node:internal/modules/helpers:136:16)
    at requireNative (/app/server/node_modules/libsql/index.js:22:10)
    at Object.<anonymous> (/app/server/node_modules/libsql/index.js:45:5)
    at Module._compile (node:internal/modules/cjs/loader:1562:14)
    at Object..js (node:internal/modules/cjs/loader:1699:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/server/node_modules/libsql/index.js' ]
}
```

could you provide packages for armv6 and armv7?

--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -2,3 +2,5 @@
 rustflags = ["-C", "target-feature=-crt-static"]
 [target.aarch64-unknown-linux-musl]
 rustflags = ["-C", "target-feature=-crt-static"]
+[target.arm-unknown-linux-musleabihf]
+rustflags = ["-C", "target-feature=-crt-static"]

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -107,7 +107,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl, arm-unknown-linux-gnueabihf]
+        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl, arm-unknown-linux-gnueabihf, arm-unknown-linux-musleabihf]
     runs-on: ubuntu-latest
     permissions:
       contents: write

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -107,7 +107,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl]
+        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl, arm-unknown-linux-gnueabihf]
     runs-on: ubuntu-latest
     permissions:
       contents: write

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -84,7 +84,7 @@ jobs:
     name: Builds (other platforms)
     strategy:
       matrix:
-        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl]
+        target: [x86_64-unknown-linux-gnu, x86_64-unknown-linux-musl, aarch64-unknown-linux-gnu, aarch64-unknown-linux-musl, arm-unknown-linux-gnueabihf, arm-unknown-linux-musleabihf]
     runs-on: ubuntu-latest
     permissions:
       contents: write

--- a/index.js
+++ b/index.js
@@ -1,7 +1,7 @@
 "use strict";
 
 const { load, currentTarget } = require("@neon-rs/load");
-const { familySync, GLIBC } = require("detect-libc");
+const { familySync, GLIBC, MUSL } = require("detect-libc");
 
 function requireNative() {
   if (process.env.LIBSQL_JS_DEV) {
@@ -11,13 +11,17 @@ function requireNative() {
   // Workaround for Bun, which reports a musl target, but really wants glibc...
   if (familySync() == GLIBC) {
     switch (target) {
-    case "linux-x64-musl":
-      target = "linux-x64-gnu";
-      break;
-    case "linux-arm64-musl":
-      target = "linux-arm64-gnu";
-      break;
+      case "linux-x64-musl":
+        target = "linux-x64-gnu";
+        break;
+      case "linux-arm64-musl":
+        target = "linux-arm64-gnu";
+        break;
     }
+  }
+  // @neon-rs/load doesn't detect arm musl
+  if (target === "linux-arm-gnueabihf" && familySync() == MUSL) {
+      target = "linux-arm-musleabihf";
   }
   return require(`@libsql/${target}`);
 }

--- a/package.json
+++ b/package.json
@@ -10,7 +10,8 @@
   "cpu": [
     "x64",
     "arm64",
-    "wasm32"
+    "wasm32",
+    "arm"
   ],
   "main": "index.js",
   "types": "types/index.d.ts",
@@ -50,7 +51,9 @@
       "x86_64-apple-darwin": "@libsql/darwin-x64",
       "x86_64-pc-windows-msvc": "@libsql/win32-x64-msvc",
       "x86_64-unknown-linux-gnu": "@libsql/linux-x64-gnu",
-      "x86_64-unknown-linux-musl": "@libsql/linux-x64-musl"
+      "x86_64-unknown-linux-musl": "@libsql/linux-x64-musl",
+      "arm-unknown-linux-gnueabihf": "@libsql/linux-arm-gnueabihf",
+      "arm-unknown-linux-musleabihf": "@libsql/linux-arm-musleabihf"
     }
   },
   "repository": {

--- a/promise.js
+++ b/promise.js
@@ -1,7 +1,7 @@
 "use strict";
 
 const { load, currentTarget } = require("@neon-rs/load");
-const { familySync, GLIBC } = require("detect-libc");
+const { familySync, GLIBC, MUSL } = require("detect-libc");
 
 // Static requires for bundlers.
 if (0) {

--- a/promise.js
+++ b/promise.js
@@ -25,13 +25,17 @@ function requireNative() {
   // Workaround for Bun, which reports a musl target, but really wants glibc...
   if (familySync() == GLIBC) {
     switch (target) {
-    case "linux-x64-musl":
-      target = "linux-x64-gnu";
-      break;
-    case "linux-arm64-musl":
-      target = "linux-arm64-gnu";
-      break;
-    }
+      case "linux-x64-musl":
+        target = "linux-x64-gnu";
+        break;
+      case "linux-arm64-musl":
+        target = "linux-arm64-gnu";
+        break;
+      }
+  }
+  // @neon-rs/load doesn't detect arm musl
+  if (target === "linux-arm-gnueabihf" && familySync() == MUSL) {
+      target = "linux-arm-musleabihf";
   }
   return require(`@libsql/${target}`);
 }

